lib/bibliothecary/parsers/maven.rb

```diff
@@ -12,6 +12,8 @@ class Maven
       # "|    \\--- com.google.guava:guava:23.5-jre (*)"
       GRADLE_DEP_REGEX = /(\+---|\\---){1}/
 
+      MAVEN_PROPERTY_REGEX = /^(.*)\$\{(.+)\}(.*)/
```

Comment:
I think `/\$/{(.+?)\}/` would do here

Reply:
It wouldn't capture any properties in the middle of the string, which was one of the problems we were having.

Reply:
That was due to the `^` marking "beginning of line"